src/components/split-button/split-button.test.js

```diff
@@ -499,3 +478,64 @@ context("Tests for Split Button component", () => {
     });
   });
 });
+
+describe("check accessibility for Split Button component", () => {
+  it("should pass accessibility tests for Split Button Default story", () => {
```

Comment:
I see we check accessibility on the `Default` story when the additional buttons are opened, as well as shut, which is great 👍 Just wondering if we should do the same on each of the other stories - at the moment you're only testing them when initially rendered, so presumably closed. Not saying we definitely should - it may be overkill, as these additional props shouldn't change the behaviour of the open buttons much or at all. But if we only test one way it might be better to test with them open than closed?

I'm not sure what the agreed QA practice is on this, feel free to ignore this!

Reply:
Yes, those are all good points. I realised shortly after I left this comment that this would still get QA reviewers, so I'm happy to wait to see what they say and go with the QA concensus 😄

Reply:
More tests added for primary type and hovering on both sections of primary and secondary type buttons.

Reply:
I agree with @sianford , those `accessibility` are enough IMO